scenarios/csharp/dotnet/captioning/Program.cs

```diff
@@ -0,0 +1,353 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+// Notes:
+// - Install the Speech NuGet package:
+// dotnet add package Microsoft.CognitiveServices.Speech
+// - The Speech SDK on Windows requires the Microsoft Visual C++ Redistributable for Visual Studio 2019 on the system. See:
+// https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-sdk
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.CognitiveServices.Speech.Audio;
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
@v-jaswel - The build fails for me without this using statement
// File
using System.IO;

Reply:
Thank you, added. Will resolve this when I push.